code/game/objects/effects/spawners/costumespawner.dm

```diff
@@ -0,0 +1,144 @@
+/obj/effect/landmark/costume/chicken/New()
+	new /obj/item/clothing/suit/chickensuit(src.loc)
+	new /obj/item/clothing/head/chicken(src.loc)
+	new /obj/item/reagent_containers/food/snacks/egg(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/gladiator/New()
+	new /obj/item/clothing/under/gladiator(src.loc)
+	new /obj/item/clothing/head/helmet/gladiator(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/madscientist/New()
+	new /obj/item/clothing/under/gimmick/rank/captain/suit(src.loc)
+	new /obj/item/clothing/head/flatcap(src.loc)
+	new /obj/item/clothing/suit/storage/labcoat/mad(src.loc)
+	new /obj/item/clothing/glasses/gglasses(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/elpresidente/New()
+	new /obj/item/clothing/under/gimmick/rank/captain/suit(src.loc)
+	new /obj/item/clothing/head/flatcap(src.loc)
+	new /obj/item/clothing/mask/cigarette/cigar/havana(src.loc)
+	new /obj/item/clothing/shoes/jackboots(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/nyangirl/New()
+	new /obj/item/clothing/under/schoolgirl(src.loc)
+	new /obj/item/clothing/head/kitty(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/maid/New()
+	new /obj/item/clothing/under/blackskirt(src.loc)
+	var/CHOICE = pick( /obj/item/clothing/head/beret , /obj/item/clothing/head/rabbitears )
+	new CHOICE(src.loc)
+	new /obj/item/clothing/glasses/sunglasses/blindfold(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/butler/New()
+	new /obj/item/clothing/suit/wcoat(src.loc)
+	new /obj/item/clothing/under/suit_jacket(src.loc)
+	new /obj/item/clothing/head/that(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/scratch/New()
+	new /obj/item/clothing/gloves/color/white(src.loc)
+	new /obj/item/clothing/shoes/white(src.loc)
+	new /obj/item/clothing/under/scratch(src.loc)
+	if(prob(30))
+		new /obj/item/clothing/head/cueball(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/highlander/New()
+	new /obj/item/clothing/under/kilt(src.loc)
+	new /obj/item/clothing/head/beret(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/prig/New()
+	new /obj/item/clothing/suit/wcoat(src.loc)
+	new /obj/item/clothing/glasses/monocle(src.loc)
+	var/CHOICE= pick( /obj/item/clothing/head/bowlerhat, /obj/item/clothing/head/that)
+	new CHOICE(src.loc)
+	new /obj/item/clothing/shoes/black(src.loc)
+	new /obj/item/cane(src.loc)
+	new /obj/item/clothing/under/sl_suit(src.loc)
+	new /obj/item/clothing/mask/fakemoustache(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/plaguedoctor/New()
+	new /obj/item/clothing/suit/bio_suit/plaguedoctorsuit(src.loc)
+	new /obj/item/clothing/head/plaguedoctorhat(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/nightowl/New()
+	new /obj/item/clothing/under/owl(src.loc)
+	new /obj/item/clothing/mask/gas/owl_mask(src.loc)
+	qdel(src)
+
+/obj/effect/landmark/costume/waiter/New()
+	new /obj/item/clothing/under/waiter(src.loc)
+	var/CHOICE= pick( /obj/item/clothing/head/kitty, /obj/item/clothing/head/rabbitears)
```

Comment:
Spessing after `CHOICE` ;)

Reply:
Yeh, I didn't write any of that code, I just moved it into its own file. Kinda feel like the costume landmarks are out of scope, want to keep the already creeping scope down.